response_operations_ui/controllers/uaa_controller.py

```diff
@@ -50,3 +50,61 @@ def sign_in(username, password):
     except (JSONDecodeError, ValueError):
         logger.exception("Error decoding JSON response")
         abort(500)
+
+
+def login_admin():
+    headers = {'Content-Type': 'application/x-www-form-urlencoded',
+               'Accept': 'application/json'}
+    payload = {'grant_type': 'client_credentials',
+               'response_type': 'token',
+               'token_format': 'opaque'}
+    try:
+        url = '{}/oauth/token'.format(app.config['UAA_SERVICE_URL'])
+        response = requests.post(url, headers=headers, params=payload,
+                                 auth=(app.config['UAA_CLIENT_ID'], app.config['UAA_CLIENT_SECRET']))
+        resp_json = response.json()
+        return resp_json.get('access_token')
+    except HTTPError:
+        logger.exception(f'Failed to log into UAA', status_code=response.status_code)
+        abort(response.status_code)
+
+
+def get_user_by_email(email):
+    headers = {'Content-Type': 'application/json',
+               'Accept': 'application/json',
+               'Authorization': 'Bearer {}'.format(login_admin())}
+
+    url = '{0}/Users?filter=email+eq+%22{1}%22'.format(app.config['UAA_SERVICE_URL'], email)
+    response = requests.get(url, headers=headers)
+    if response.status_code != 200 or response.json()['totalResults'] == 0:
+        pass
```

Comment:
This pass does not do anything ? Not sure what the intention was , did you mean return ?

Reply:
I did!